File: WEEK1/seminar/src/main/java/com/server/dosopt/seminar/dto/HealthCheckResponse.java
package com.server.dosopt.seminar.dto;

public class HealthCheckResponse {

    private String status;
    private final String STATUS_OK = "OK";

    public HealthCheckResponse(String status) {
        this.status = status;
    }

    public HealthCheckResponse() {
        this.status = STATUS_OK;
    }
}

File: kotlin-seminar/src/main/kotlin/sopt/org/kotlinseminar/common/response/HealthCheckResponse.kt
package sopt.org.kotlinseminar.common.response

class HealthCheckResponse(
    var status: String
) {

    val STATUS_OK = "OK"   // 타입을 생략해도 컴파일러의 분석으로 String으로 인식

    fun HealthCheckResponse() {
        this.status = STATUS_OK;
    }
}